src/LambdasClient.ts

```diff
@@ -36,6 +36,9 @@ export class LambdasClient implements LambdasAPI {
   }
 
   fetchProfiles(ethAddresses: string[], profileOptions?: ProfileOptions, options?: RequestOptions): Promise<any[]> {
+    if (ethAddresses.length === 0) {
+      throw new Error('At least one id must be present when requesting profiles')
```

Comment:
Return [] instead?

Reply:
yes, better!

Reply:
Should we refactor all the lambdas client then? https://github.com/decentraland/catalyst-client/blob/22248f778bf7983bfcd00882881d2fadc1e37e6c/src/LambdasClient.ts#L68

🤔

Reply:
I'd say if applied consistently, that approach is safer. Maybe it doesn't need to change everywhere at once tho